### ISSUE TITLE
Add undocumented update switches to readme

### ISSUE THREAD

**Issue by AndrewFinlay** [angular/webdriver-manager#235]
Hi there,

After having a look through the code and comments for webdriver-manager, I realised that most of my problems would magically go away if I new about the --versions.chrome update switch.  Using this switch and the listing in the Chromedriver repo I can download the latest driver manager and test against the latest and greatest Chrome release.  So I propose that it gets documented along with the limitations in the current/legacy implementations of the update command.

I'd do this myself and submit a pull request but I'm unfamiliar with how this process works for browsers other than Chrome.  I figure there may also be risks involved in using a driver too far in advance of the webdriver-manager version.

Anyway let me know what you think about a documentation based fix.

Regards,
Andrew

**Comment by cnishina** [comment]
Sure this sounds reasonable.

**Comment by kgentes** [comment]
While I would simply defer to a "me too" to get a complete list of options added to the documentation, one thing that may help folks with this documentation lack temporarily is the list of  `webdriver-manager update` options is already available using the "--help" switch.  As in `webdriver-manager update --help` .

This results in :

```
Usage:        update [options]
              update help
Description:  install or update selected binaries

Options:
  --out_dir                   Location to output/expect                                                                              [default: C:\git\test-batch-designer\node_modules\protractor\node_modules\webdriver-manager\selenium]
  --verbose                   Extra console output
  --ignore_ssl                Ignore SSL certificates
  --proxy                     Proxy to use for the install or update command
  --alternate_cdn             Alternate CDN to binaries
  --standalone                Install or update selenium standalone                                                                  [default: true]
  --chrome                    Install or update chromedriver                                                                         [default: true]
  --gecko                     Install or update geckodriver                                                                          [default: true]
  --android                   Update/use the android sdk
  --android-api-levels        Which versions of the android API you want to emulate                                                  [default: 24]
  --android-archs             Which architectures you want to use in android emulation.  By default it will try to match os.arch()   [default: x86_64]
  --android-platorms          Which platforms you want to use in android emulation                                                   [default: google_apis]
  --android-accept-licenses   Automatically accept android licenses
  --ie                        Install or update 32-bit ie driver
  --ie32                      Install or update 32-bit ie driver
  --ie64                      Update: install or update 64-bit IE driver. Start: use installed x64 IE driver.
  --versions.standalone       Optional seleniuim standalone server version (use 'latest' to get the most recent version)             [default: latest]
  --versions.chrome           Optional chrome driver version (use 'latest' to get the most recent version)                           [default: latest]
  --versions.appium           Optional appium version                                                                                [default: 1.6.0]
  --versions.android          Optional android sdk version                                                                           [default: 24.4.1]
  --versions.gecko            Optional gecko driver version                                                                          [default: latest]
  --versions.ie               Optional internet explorer driver version (use 'latest' to get the most recent version)                [default: latest]

```

**Comment by cnishina** [comment]
@kgentes You've found the super secret menu that has zero documentation.

**Comment by cnishina** [comment]
Closed! with PR #241. Docs should be viewable from the readme.
